server/src/game/mod.rs

```diff
@@ -74,7 +75,7 @@ pub struct Game {
     pub mafia: Mafia,
     pub cult: Cult,
     pub arsonist_doused: ArsonistDoused,
-    // pub cleaned: Cleaned,
+    // pub cremated: Cremated,
```

Comment:
Why not implement this while you're at it? You'll have to do it eventually, plus otherwise this is just a rename commit

Reply:
Hm, actually, why have this at all? Unless you want different morticians to share tagged players... Idk

Reply:
Comment was removed